### PR TITLE
[FIX]remove stock from depends

### PR DESCRIPTION
**Pull request by pledrateam**
remove stock from depends of base module which is not needed.

### CODE REVIEW

**Review by PCatinean — APPROVED** [review]

File: product_configurator/__manifest__.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "author": "Pledra, Odoo Community Association (OCA)",
     "license": "AGPL-3",
     "website": "https://github.com/OCA/product-configurator",
-    "depends": ["account", "stock"],
+    "depends": ["account"],
```
Comment (dreispt) [inline product_configurator/__manifest__.py:9]: Shouldn't it at least depend from `product`?

Comment (PCatinean) [reply product_configurator/__manifest__.py:9]: It's included in the chain, account depends on product https://github.com/odoo/odoo/blob/13.0/addons/account/__manifest__.py#L18

Comment (dreispt) [reply product_configurator/__manifest__.py:9]: OK. 
I'm totally for lighter dependencies. 
Still, Travis build is red, so something else is probably missing.

**Comment by bizzappdev** [comment]
one view change was there which was depending upon the stock module.
we will take care while migrating stock module to move that view related changes to stock module itself.

**Review by dreispt — APPROVED** [review]

**Pull request merged**